webapp/test/data/cache.yml

```diff
@@ -382,6 +390,7 @@ repo_detail:
   - null
   - null
   - '2019-09-20T06:25:12+00:00'
+  - false
```

Comment:
All repos are third_party false here. Then it's hard to test whether new feature works properly. Could you add some third_party repo and update tests accordingly?

Reply:
Added some test data for advisories and repos.